config/config.go

```diff
@@ -88,6 +88,8 @@ type Configuration struct {
 	GenerateBidID bool `mapstructure:"generate_bid_id"`
 	// GenerateRequestID overrides the bidrequest.id in an AMP Request or an App Stored Request with a generated UUID if set to true. The default is false.
 	GenerateRequestID bool `mapstructure:"generate_request_id"`
+	// Integration Type value lives in config so that it can be updated from the request, and then passed for the event URL
+	IntegrationType string
```

Comment:
It's because I was having trouble getting the integration information to update the vTrack URL integration parameter to the cachePutObjects function.  The cachePutObjects function, which eventually calls EventRequestToUrl, has the config object as a member of its function, so by adding the IntegrationType value to the config object, it allowed me to then access it to update the vtrack URL.

This was confusing for me to type, so it might be confusing for you to read, so we can also talk about it tomorrow on a call if you'd like to see if there's an easier way for me to get the integration type value to the vtrack URL.